src/Message/FetchTransactionRequest.php

```diff
@@ -11,6 +11,8 @@
      */
     public function getData()
     {
+        $this->validate('transactionReference');
```

Comment:
There is currently no part of the public interface which advertises this exception, which is an issue we might raise in omnipay/common. It is the correct place however, all other omnipay packages place validation here as well.